main.go

```diff
@@ -39,12 +42,22 @@ type data struct {
 
 func value(output tfjson.StateOutput) string {
 	if output.Sensitive {
-		return "sensitive; redacted"
+		return sensitive
 	}
 
 	return fmt.Sprintf("%v", output.Value)
 }
 
+func prettyPrintValue(output tfjson.StateOutput) template.HTML {
+	if output.Sensitive {
+		return template.HTML(sensitive)
+	}
+
+	pretty, _ := json.MarshalIndent(output.Value, "", "  ")
+
+	return template.HTML(string(pretty))
```

Comment:
By returning a `template.HTML`, I avoid escaping quotes and littering the output with `&#34;`. I think I'd prefer to avoid escaping the quotes, especially because this could present challenges for those users who attempt to publish the output to release notes sent via GitHub API requests whose JSON payload contains the escaped output. In those cases, if the output contained characters such as `&#34;`, I believe they'd end up double escaped.